### ISSUE TITLE
rancher2 CLI - should support environment variable for cli2.json config

### ISSUE THREAD

**Issue by PicoCreator** [rancher/rancher#15195]
Currently the `cli2.json` file is hard-coded to the user home directory (see : https://github.com/rancher/cli/search?q=cli2.json&unscoped_q=cli2.json)

For our use case, we tend to have multiple environment we managed in v1.6 in different folders, with a bash script overwriting the required environment variables for login. This allows us to simultaneously interact with multiple environments without "logging in and out" of contexts.

A rather hackish work around that I do now with a bash script

```bash
$( export HOME="$WORKDIR" && ./rancher "$@" )
```

Where it will take the current configured WORKDIR/.rancher/cli2.json file instead.
Ideally it should be something like this instead

```bash
./rancher --config="./cli2.json" "$@"
```

**Comment by benzman81** [comment]
need this for parallel execution, too. Each call can be a different server/context, so parallel execution is not possible by now.

**Comment by benzman81** [comment]
> A rather hackish work around that I do now with a bash script
> 
> ```shell
> export HOME="$WORKDIR"; 
> ./rancher "$@"
> ```

This work around works for me at the moment.

**Comment by mkjpryor-stfc** [comment]
This would be great. The fixed config location is a PITA for using the Rancher CLI in CI workflows.

**Comment by sumanthkumarc** [comment]
Agreed with above statement. It would be good to provide an alternative using env or another json file.

**Comment by miraculixx** [comment]
This seems to have been addressed in https://github.com/rancher/cli/commit/5860516f5e866a41e45b49b50d1168a3c58b234d

**Comment by PicoCreator** [comment]
@miraculixx can I confirm the syntax would be something like the following for version `v2.4.11-rc2` onwards? (if so I can help close this issue)

```
./rancher --config="./cli2.json" <your command here>
```
